extensions/database/src/com/google/refine/extension/database/DatabaseModuleImpl.java

```diff
@@ -65,18 +65,37 @@ public void init(ServletConfig config)
         logger.trace("Database Extension module initialization completed");
     }
 
-    public static String getImportCreateBatchSize() {
-        if (extensionProperties == null) {
-            return DEFAULT_CREATE_PROJ_BATCH_SIZE;
-        }
-        return extensionProperties.getProperty("create.batchSize", DEFAULT_CREATE_PROJ_BATCH_SIZE);
+    public static int getCreateBatchSize() {
+        return getBatchSize("create.batchSize", DEFAULT_CREATE_BATCH_SIZE);
+    }
+
+    public static int getPreviewBatchSize() {
+        return getBatchSize("preview.batchSize", DEFAULT_PREVIEW_BATCH_SIZE);
     }
 
-    public static String getImportPreviewBatchSize() {
+    private static int getBatchSize(String propertyName, String defaultValue) {
+        int batchSize = Integer.parseInt(defaultValue);
+
         if (extensionProperties == null) {
-            return DEFAULT_PREVIEW_BATCH_SIZE;
+            return batchSize;
+        }
+
+        String propBatchSize = extensionProperties.getProperty(propertyName, defaultValue);
```

Comment:
I think we should be able to remove the default value here. The if statement on the next line should still be fine since `getProperty` will still return null if there's no value.